client/scripts/views/components/header/login_selector.ts

```diff
@@ -127,10 +128,12 @@ const LoginSelector: m.Component<{
       ]),
     ]);
 
+    // TODO: figure out where this gets populated when page is loaded
```

Comment:
Look into `login.ts`, maybe the `updateActiveAddresses` function.

Reply:
Oops, meant to delete comments like this...

Reply:
should still do :)